Lib/inspect.py

```diff
@@ -3107,6 +3128,17 @@ def from_callable(cls, obj, *,
                                         follow_wrapper_chains=follow_wrapped,
                                         globals=globals, locals=locals, eval_str=eval_str)
 
+    @classmethod
+    def from_code(cls, co):
+        """Constructs Signature for the given code object.
```

Comment:
(No change requested)

Generally we prefer the imperative mood for docstrings, but this is in keeping with the style of the other docstrings in this class, so I think it works well 👍

Reply:
The imperative meaning:
```suggestion
        """Construct Signature for the given code object.
```

Reply:
> The imperative meaning

Indeed, but, as I say, all other constructor methods on this class currently use "Constructs" rather than "Construct". I think consistency within the class is more important here, so I prefer the wording that @sobolevn already has.

Reply:
I used the same style as in `from_callable`